Source/Scene/Expression.js

```diff
@@ -66,20 +66,43 @@ define([
                 }
             }
         } else if (ast.type === 'UnaryExpression') {
-            var op = ast.operator;
+            op = ast.operator;
             var child = createRuntimeAst(expression, ast.argument);
-            if (op === '!') {
-                node = new Node(op, child);
-            } else if (op === '-') {
+            if (op === '!' || op === '-') {
                 node = new Node(op, child);
             }
+        } else if (ast.type === 'BinaryExpression') {
+            op = ast.operator;
+            var left = createRuntimeAst(expression, ast.left);
+            var right = createRuntimeAst(expression, ast.right);
+            if (op === '+' || op === '-' || op === '*' ||
```

Comment:
It can a future PR, but I'm not sure how much father we should go without introducing error checking for syntax errors (errors to us, not just jsep).  I'm afraid it will be increasingly painful to add later.